### ISSUE TITLE
User Profile Icon - Color Contrast Issue

### ISSUE THREAD

**Issue by vishnunavuluri** [US-EPA-CAMD/PI-1#2515]
The color contrast is failing on the User Profile icon. Update the colors as below:

Icon Background: White (#FFFFFF)
User Initials: Blue base (#005EA2)

Reference:
RF-1b 
from Report https://app.zenhub.com/files/287570343/c84cab03-f68f-421a-8476-705e237a8283/download
**Updated Prototype (UserID Icon): https://xd.adobe.com/view/28b8b774-a061-44ae-9a20-e9f1089de84f-7deb/screen/97676703-53f4-4741-9753-c28998f50fa9/specs/

**Comment by mosesdeeCVP** [comment]
![2515 User Profile Icon - Color Contrast Issue.png](https://images.zenhubusercontent.com/5f807025564a417cbcdf6d69/8a0597b2-4317-4e60-9832-9639a7a48f3f)

**Comment by mosesdeeCVP** [comment]
Bug fixed
